src/Integration/Laravel/Filesystem/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## NOT RELEASED
 
+## 0.2.2
+
+### Fixed
+
+- Fix laravel locking to use consistent reads
```

Comment:
This changelog belongs to src/Integration/Laravel/Cache/CHANGELOG.md

Reply:
good catch !!